charts/argo/values.yaml

```diff
@@ -115,6 +115,7 @@ controller:
   # service type `LoadBalancer`
   loadBalancerSourceRanges: []
   resources: {}
+  env: []
```

Comment:
Might pay to put an example of what extraEnv could look like.

Reply:
These values serve as defaults so they are not an appropriate place to put examples. I've added a description in comment instead.